cmd/snap-device-helper/snap-device-helper.c

```diff
@@ -127,16 +160,59 @@ static char *udev_to_security_tag(const char *udev_tag) {
         die("missing snap name in tag \"%s\"", udev_tag);
     }
 
+    char *snap_component = NULL;
```

Comment:
otherwise it's confusing as snap component is `<snap>+<name>`, but component name is just `<name>`
```suggestion
    char *snap_component_name = NULL;
```
or just `component_name` and `component_name_buffer` ?